suzieq/poller/controller/source/ansible.py

```diff
@@ -85,8 +85,9 @@ def _get_inventory(self) -> Dict:
                 password = entry['ansible_password']
 
             # Retrieve password information
-            devtype = entry.get('ansible_network_os', None)
-            if devtype == 'eos':
+            devtype = None
+            if entry.get('ansible_network_os') in ['eos', 'panos']:
+                devtype = entry.get('ansible_network_os')
```

Comment:
The correct fix is to check for https and only use devtype if the transport is https. The second thing is to allow device autodetection to override the devtype. This fix will have to be redone.

Reply:
We restored the ansible source to the version which is now used on the 0.15 a part from the 'panos' check. I don't get what you mean with 'allow device autodection'

Reply:
We use ssh to autodetect a device type. If that provides an answer, don't override it with whatever Ansible provides.